src/utils/data/applicationFilters.ts

```diff
@@ -0,0 +1,128 @@
+import { DocumentNode } from 'graphql'
+import {
+  ApplicationOutcome,
+  ApplicationStatus,
+  AssignerAction,
+  ReviewerAction,
+} from '../generated/graphql'
+import getApplicantFilterList from '../graphql/queries/applicationListFilters/getApplicantFilterList'
+import getAssignerFilterList from '../graphql/queries/applicationListFilters/getAssignerFilterList'
+import getOrganisationFilterList from '../graphql/queries/applicationListFilters/getOrganisationFilterList'
+import getReviewersFilterList from '../graphql/queries/applicationListFilters/getReviewersFilterList'
+import getStageFilterList from '../graphql/queries/applicationListFilters/getStageFilterList'
+import { FilterDefinitions, GetFilterListQuery } from '../types'
+
+const constructFilterListQuery = (query: DocumentNode, queryMethod: string) => {
+  const getListQuery: GetFilterListQuery = ({ searchValue, filterListParameters }) => {
+    const { templateCode = '' } = filterListParameters || {}
+    return {
+      query,
+      variables: {
+        searchValue,
+        templateCode,
+      },
+      resultExtractor: (resultQuery) => ({
+        list: resultQuery?.[queryMethod]?.nodes || [],
+        totalCount: resultQuery?.[queryMethod]?.totalCount || 0,
+      }),
+    }
+  }
+  return getListQuery
+}
+
+export const APPLICATION_FILTERS: FilterDefinitions = {
+  lastActiveDate: {
+    type: 'date',
+    title: 'check',
+  },
+  isFullyAssignedLevel1: {
+    type: 'boolean',
+    title: 'Assignment Status',
+    options: {
+      booleanMapping: {
+        true: 'Fully Assigned',
+        false: 'Not Fully Assigned',
+      },
+    },
+  },
+  assignReviewerAssignedCount: {
```

Comment:
Why not `reviewersAssignedCount` instead? I haven't really understood what is this filter though. Not a visible one so only available on the URL correct?

Reply:
yeah, not sure what it's for either.. it was there so decided to expose it (since there are no other number filters, so used this one as a template for future number filters)
Actually I can't see it in the list at all now, so commented it out: d49e730902ff11f538f39fa8e6de66cf4f06769f